shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -11,18 +11,19 @@
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterEngine_Internal.h"
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterPluginAppLifeCycleDelegate_internal.h"
 
+FLUTTER_ASSERT_ARC
+
 static NSString* const kUIBackgroundMode = @"UIBackgroundModes";
 static NSString* const kRemoteNotificationCapabitiliy = @"remote-notification";
 static NSString* const kBackgroundFetchCapatibility = @"fetch";
 static NSString* const kRestorationStateAppModificationKey = @"mod-date";
 
 @interface FlutterAppDelegate ()
 @property(nonatomic, copy) FlutterViewController* (^rootFlutterViewControllerGetter)(void);
+@property(nonatomic, strong) FlutterPluginAppLifeCycleDelegate* lifeCycleDelegate;
 @end
 
-@implementation FlutterAppDelegate {
-  FlutterPluginAppLifeCycleDelegate* _lifeCycleDelegate;
```

Comment:
Looks like we are favoring properties over ivars. It may make sense to use dot syntax as well in the caller as well (self.lifeCycleDelegate in line 37). 

I'm ok with either, but better to be consistent.

Reply:
Ah good call; done -- other than in the initialiser where `self` isn't fully initialised.